superset/email.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import
```

Comment:
Can we just squeeze these functions in utils.py? Looks like modules have been multiplying in the superset namespace. Let's keep the main namespace clean.

I don't think utils.py is big yet, but if it gets there we can create a utils folder and break down the utilities in different modules